ballerina/listener_utils.bal

```diff
@@ -355,3 +364,12 @@ isolated function getHttpServiceFromGraphqlService(Service s) returns HttpServic
 @java:Method {
     'class: "io.ballerina.stdlib.graphql.runtime.engine.ListenerUtils"
 } external;
+
+isolated function attachUpgradeServiceToGraphqlService(Service s, UpgradeService wsService) = @java:Method {
```

Comment:
Shouldn't this be `attachWebsocketServiceToGraphqlService` (If we follow that same pattern)?